pybossa/cache/site_stats.py

```diff
@@ -421,3 +421,53 @@ def management_dashboard_stats_cached():
         # reset stats for any missing stat
         map(delete_cache_group, management_dashboard_stats)
     return stats_cached
+
+def total_tasks(interval):
```

Comment:
Maybe we should enhance the existing function ```n_tasks_site```?
Benefit:
- Reduce code duplicate
- Get cache for free (this is a slow query - we do need cache as you pointed out in the Jira ticket)